### PR TITLE
DBZ-3444 KRaft example

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.redhat.com/browse/DBZ-3444

@jpechane, as dicussed, I'd suggest you push only the first commit. I've added the second to facilitate testing in both modes, combined and non-combined.

### CODE REVIEW

File: tutorial/docker-compose-zookeeperless-kafka.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  connect:
+    image: debezium/connect:${DEBEZIUM_VERSION}
+    ports:
+      - 8083:8083
+    links:
+      - kafka-1
+      - kafka-2
+      - kafka-3
```
Comment (jpechane) [inline tutorial/docker-compose-zookeeperless-kafka.yaml:69]: Should not kafka-4 be linked too?

File: tutorial/README.md [hunk 2]
```diff
@@ -482,6 +483,34 @@ curl -s http://localhost:8083/connectors/inventory-connector/config | jq .
...
+```shell
+# Start the topology as defined in https://debezium.io/docs/tutorial/
+export DEBEZIUM_VERSION=1.7
+docker-compose -f docker-compose-zookeeperless-kafka-combined.yaml up
+
+# Start Postgres connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @register-postgres.json
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-zookeeperless-kafka-combined.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
```
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply tutorial/README.md:499]: Yes, this should be kafka-1, or any other. But it doesn't matter which one, you can use the console consumer from anywhere, it doesn't read messages only from its local broker node.

File: tutorial/docker-compose-zookeeperless-kafka.yaml (hunk 1 above)
Comment (gunnarmorling) [reply tutorial/docker-compose-zookeeperless-kafka.yaml:69]: Done.

File: tutorial/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply tutorial/README.md]: Done.

**Comment by gunnarmorling** [comment]
@jpechane, updated as requested (btw. I think quickest is if you do such tiny change yourself and apply it during review, instead of doing another circle). This PR and the other two for the AK 2.8 update should be good to go.

**Comment by jpechane** [comment]
@gunnarmorling Well, it was intentiaonal to do another round as I was unsure about the impact of partitioning on reading from one broker but it seems this is ok.

File: tutorial/README.md [hunk 3]
```diff
@@ -482,6 +483,34 @@ curl -s http://localhost:8083/connectors/inventory-connector/config | jq .
...
+# Start the topology as defined in https://debezium.io/docs/tutorial/
+export DEBEZIUM_VERSION=1.7
+docker-compose -f docker-compose-zookeeperless-kafka-combined.yaml up
+
+# Start Postgres connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @register-postgres.json
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-zookeeperless-kafka-combined.yaml exec kafka-1 /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
```
Comment (jpechane) [inline tutorial/README.md:500]:
```suggestion
    --bootstrap-server kafka-1:9092 \
```

**Comment by jpechane** [comment]
@gunnarmorling Applied, thanks for all three PRs!

**Pull request merged**